Attempt to make propTablesExclude usage clearer

mattferderer opened the pull request:
See issue #2266, specifically https://github.com/storybooks/storybook/issues/2266#issuecomment-351472412 for discussion.

Issue: #2266 

## What I did
Added text to documentation make propTablesExclude clearer

## How to test
No changes to code made.

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
No but would maybe be a good idea.

Does this need an update to the documentation?
This is an update.

If your answer is yes to any of these, please make sure to include it in your PR.

mattferderer committed:
Attempt to make propTablesExclude usage clearer

See issue #2266, specifically https://github.com/storybooks/storybook/issues/2266#issuecomment-351472412 for discussion.

Hypnosphi commented on addons/info/README.md:
```diff
@@ -130,7 +130,7 @@ setDefaults({
   source: true, // Displays the source of story Component
   propTables: [/* Components used in story */], // displays Prop Tables with this components
-  propTablesExclude: [], // Exclude Components from being shown in Prop Tables section
+  propTablesExclude: [], // Exclude Components from being shown in Prop Tables section. Pass the imported component names as values to the array.
```
Component name is a string, but here the component itself (class or stateless function) should be passed

mattferderer commented:
Would it make more sense to say "Pass the imported component class or function to the array."

Hypnosphi commented:
"Accepts an array of component classes or functions"

mattferderer committed:
Merge branch 'master' into patch-2

mattferderer committed:
Updated Readme.md propTablesExclude

Changed to agreed upon instructions for propTablesExclude to provide clearer documentation.

Hypnosphi committed:
Merge branch 'master' into patch-2
